### PR TITLE
Update EIP-5585: Remove staking functions and move to Review

### PR DESCRIPTION
**Pull request by featheryus**
Remove staking functions and change the EIP status to Review.

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-5585.md [hunk 1]
```diff
@@ -163,7 +159,7 @@ This standard is compatible with [EIP-721](./eip-721.md) since it is an extensio
 
 ## Security Considerations
 
-If someone buys an NFT within the duration of an authorization, they will not have to stake anything, providing no incentive to cancel the authorization.
+If a NFT owner sold his NFT with an authorization, we need to make sure the new onwer can have the rest authorization fee and will not cancel the authorization.
```
Comment (Pandapip1) [inline EIPS/eip-5585.md:162]: Not sure about this wording. Might be worth rephrasing.

**Review by Pandapip1 — APPROVED** [review]
One nit. Otherwise LGTM.

File: EIPS/eip-5585.md (hunk 1 above)
Comment (VeegaLabsOfficial) [reply EIPS/eip-5585.md:162]: I have rephrased those words, thanks @Pandapip1

**Review by fovcao — DISMISSED** [review]

**Review by fovcao — APPROVED** [review]

**Review by fovcao — DISMISSED** [review]

**Review by fovcao — APPROVED** [review]

**Comment by VeegaLabsOfficial** [comment]
Hi, @Pandapip1 it seems need your further approve? Or something else I need to do?

**Comment by Pandapip1** [comment]
I'm not sure why it requires review from us.

@eth-bot rerun

**Review by Pandapip1 — APPROVED** [review]
Oh duh, you're moving it to review.

**Pull request merged**